src/Types/formFields.ts

```diff
@@ -0,0 +1,5 @@
+export interface FormFields {
+  [key: string]: string;
```

Comment:
[CODE STYLE] - The use of an index signature here is unnecessary and could lead to potential type-safety issues. Since you have already defined specific fields (title, user), you should remove the index signature to ensure that only the fields you want to be part of the interface can be used.